designer/public/js/ldlostchild.js

```diff
@@ -0,0 +1,857 @@
+$(function() {
```

Comment:
all the code is written wrt NGSI-V1 which is not as per design . Need to update as per the design(protocol independent).